src/Pages/ArticlePage.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState, useEffect } from "react"
+import { useParams } from "react-router-dom"
+import { formatDistanceToNow } from "date-fns"
+import { getArticleById } from "../api"
+
+const ArticlePage = () => {
+    const {id} = useParams()
+    const [currentArticle, setCurrentArticle] = useState({})
+    const [loading, setLoading] = useState(true)
+
+    useEffect(() => {
+        getArticleById(id)
+        .then((res) => {
+            setCurrentArticle(res)
+            setLoading(false)
+        }).catch(console.log)
+    }, [])
+console.log(currentArticle)
+    function findTimeSince(originalDate){
```

Comment:
Get rid of this `console.log` 🙅‍♀️ (line 18)

Reply:
I knew there would be one hidden somewhere lol